src/main/io/dataEdition.c

```diff
@@ -0,0 +1,1359 @@
+			    editionLine = selectedLine ;
+		        beeperConfirmationBeeps(1);
+			}
+			break;
+		}
+	}
+}
+
+static void processInterfaceOfThePage(uint8_t commandID)
+{
+	if (editionLine == -1){
+	    processEventsForLineSelection(commandID);
+		if (editionLine != -1){
+			if (selectedPage == SELECTION_PAGE){
+	            selectedPage = selectedLine + (whichOfPages * DISPLAY_LINES_MINUS1);
+	        	whichOfPages = 0;
+	        	initializeNewPage();
+			}
+			else {
+				selectedItem = firstItemOnLine(editionLine);
+				structureOfPage.editField[selectedItem].color = PIXEL_INVERTED;
+				onGoingEditionProcess = true;
+#if (DATA_EDITION_FULL_VERSION)
+				if ((selectedPage == RECEIVER_PAGE) && (selectedItem == 18)) {
+	        		structureOfPage.editField[19].data = 0;
+	        		getReceiverMapping();
+					structureOfPage.editField[selectedItem].color = PIXEL_MIXED;
+	        		structureOfPage.editField[selectedItem].data = getChannelIndex();
+				}
+#endif
+			}
+		}
+		else
+			exitEditionProcess();
+	}
+	else
+
+	switch (commandID) {
+		case INC_COMMAND : {
+			processIncDecEvent(1);
+			break;
+		}
+		case DEC_COMMAND : {
+			processIncDecEvent(-1);
+			break;
+		}
+		case NEXT_COMMAND : {
+			processNextPrevEvent (1);
+			break;
+		}
+		case PREV_COMMAND : {
+			processNextPrevEvent (-1);
+			break;
+		}
+		case SET_COMMAND : {
+#if (DATA_EDITION_FULL_VERSION)
+			if ((selectedItem == 21) && (selectedPage != MOTORS_PAGE)){
+			    saveConfigAndNotify();
+			    structureOfPage.editField[selectedItem].color = PIXEL_NORMAL;
+				exitEditionProcess();
+			}
+			else {
+				if ((selectedPage == RECEIVER_PAGE) && (selectedItem == 18)){
+					if (structureOfPage.editField[selectedItem].color == PIXEL_INVERTED)
+						structureOfPage.editField[selectedItem].color = PIXEL_MIXED;
+					else {
+						if (theRCMappingStringIsValid()){
+							parseRcChannels(tempString, &masterConfig.rxConfig);
+							structureOfPage.editField[selectedItem].color = PIXEL_NORMAL;
+							exitEditionProcess();
+						}
+						else {
+							structureOfPage.editField[selectedItem].color = PIXEL_INVERTED;
+					        beeperConfirmationBeeps(2);
+						}
+					}
+				}
+				else
+					if (onGoingEditionProcess){
+						structureOfPage.editField[selectedItem].color = PIXEL_NORMAL;
+						exitEditionProcess();
+					}
+			}
+#else
+			if (selectedItem == 21){
+				saveConfigAndNotify();
+				structureOfPage.editField[selectedItem].color = PIXEL_NORMAL;
+				exitEditionProcess();
+			}
+			else
+				if (onGoingEditionProcess){
+					structureOfPage.editField[selectedItem].color = PIXEL_NORMAL;
+					exitEditionProcess();
+				}
+#endif
+			break;
+		}
+	}
+}
+
+//=======================  end of processing interface of the page  ========================
+
+//=================================  process SelectedPage  ==================================
+static void writeSignOfSelectedLine(void)
+{
+	contentOfPage.lineOfText[selectedLine][0] = 0x3E;
```

Comment:
'>' is much easier to understand .. 
(same for all other printable ASCII codes in hex)